docs/The-Free-Plan.md

```diff
@@ -0,0 +1,70 @@
+---
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# What is the Free Plan?
+The free plan is ideal for start-ups and small businesses to manage expenses. With the Free Plan, a workspace admin can set up their team with Expensify Cards, reimburse cash expenses, send invoices, and manage bills, all for free! You will have total visibility and control over all spending associated with your workspace in real time.
+
+# Features Included with the Free Plan
+- Expensify Cards for all employees
+- Invoicing
+- Bill Pay
+- Unlimited receipt scanning for everyone in the company
+- Free next-day ACH reimbursements for cash expenses
+- Up to 1% cash back for card spending over $25,000 per month
+- Free corporate travel booking
+
+# Setting Up the Free Plan
+- Navigate to new.expensify.com, enter your company email address, and set a password
+- Click the **green “+”** button and select **_New workspace_**
+
+<img class="image" src="/assets/images/new-workspace.png">
+
+Once you’ve created your Workspace, you should receive a message from Concierge encouraging you to chat with your Setup Specialist.
+
+<img class="image" src="/assets/images/chat-with-concierge-setup.png">
+
+- Click the link sent to you by Concierge, and your designated Setup Specialist will guide you through how to configure your company setup.
+
+<img class="image" src="/assets/images/chat-with-setup-specialist.png">
+
+Once you’ve completed your company setup, you should have done the following:
+
+- Connected a business bank account (Settings menu > Click **_Connect bank account_** and follow the prompts).
+- Invited members to the workspace
+- Assigned Expensify Cards
+
+# Inviting Members to the Free Plan:
+- Navigate to the Settings Menu and click **_Manage members_** to invite your team. You can invite employees one at a time, or you can invite multiple users by listing out their email addresses separated by a comma
+- To use the Expensify Card, you must invite them to your workspace via your company email address (i.e., admin@companyemail.com and NOT admin@gmail.com).
+
+# Managing the Free Plan
+To access your workspace settings, click your profile icon and then on your workspace name:
+
+<img class="image" src="/assets/images/profile-settings.png">
+<img class="image" src="/assets/images/workspace-settings.png">
+
+This menu allows you to manage your workspace members, issue additional Expensify Cards, and utilize this plan’s various bill pay and payment options.
+
+#FAQs
+**_Do I need a business bank account to use the Free Plan?_**
```

Comment:
NAB: we can add these sub-sections as nested headers in the TOC if we convert them to the next header level (`##` is rendered as H2):

<img width="392" alt="Screen Shot 2022-10-19 at 18 48 24" src="https://user-images.githubusercontent.com/6829422/196825215-f80479a9-6dc8-4c09-8b18-f27b6d262138.png">

```suggestion
## **_Do I need a business bank account to use the Free Plan?_**
```

Reply:
Oh cool! That's great to know. Is that documented somewhere else in the README? That's pretty handy information for other folks as we write articles.

(If it's not written down, I can get it in an SO)

Reply:
Ah it's mentioned briefly [here](https://github.com/Expensify/App/tree/main/docs#add-content), but I think it could be a good idea to include it in an SO.